test/object_store/run

```diff
@@ -41,7 +41,7 @@ conn.execute("INSERT INTO test_ks.test_cf ( name, value ) VALUES ('1', 'one');")
 conn.execute("INSERT INTO test_ks.test_cf ( name, value ) VALUES ('2', 'two');")
 res = conn.execute("SELECT * FROM test_ks.test_cf;")
 
-r = requests.post(f'http://{ip}:10000/storage_service/keyspace_flush/test_ks')
+r = requests.post(f'http://{ip}:10000/storage_service/keyspace_flush/test_ks', timeout=10)
```

Comment:
Not a call to minio, but rather a call to scylla itself. When in debug build 10 seconds can be not enough, especially in loaded CI nodes. This would generate flaky test

Reply:
i see. how about 60 seconds, is it long enough ?